src/cmt_decode_prometheus.c

```diff
@@ -917,12 +932,12 @@ static int parse_histogram_summary_name(
     if (current_name_len < parsed_name_len) {
         // current name length cannot be less than the length already parsed. That means
```

Comment:
this comments were there before, pls change the commenting style with `/* .. */`